x/profiles/client/cli/query.go

```diff
@@ -175,3 +175,78 @@ func GetCmdQueryUserBlocks() *cobra.Command {
 
 	return cmd
 }
+
+// GetCmdQueryChainsLinks returns the command allowing to query all the links
+func GetCmdQueryChainsLinks() *cobra.Command {
```

Comment:
Instead of having two different commands (`GetCmdQueryChainsLinks` and `GetCmdQueryUserChainsLinks`), I think we should only have one (`GetCmdQueryUserChainsLinks`). 

This then should have an optional parameter named `[user]` that: 
- if specified, allows to query the links for such user
- if not specified, allows to query all the links for all the users.

This can be done using the `cobra.RangeArgs(0, 1)` and then checking whether the optional parameter is empty or not. You can see an example [here](https://github.com/cosmos/cosmos-sdk/blob/master/x/distribution/client/cli/query.go#L255)